packages/atomic/src/components/result-template-components/atomic-result-text/atomic-result-text.tsx

```diff
@@ -57,11 +57,15 @@ export class AtomicResultText {
   }
 
   private get resultValue() {
-    const value = ResultTemplatesHelpers.getResultProperty(
+    let value = ResultTemplatesHelpers.getResultProperty(
       this.result,
       this.field
     );
 
+    if (Array.isArray(value)) {
```

Comment:
quick improvement there too, "languages" has an array of languages, when the field is an array, we cast it to a string to allow displaying multiple string values